.github/renovate.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":dependencyDashboard",
+    ":gitSignOff",
+    ":semanticCommitScopeDisabled",
+    "schedule:earlyMondays"
+  ],
+  "prHeader": "Update Request | Renovate Bot",
+  "regexManagers": [
```

Comment:
https://github.com/siderolabs/go-api-signature/pull/8/files#diff-f82f7d36a61d22f640c25bdb8b0435bf9cb38679d99d5f460753fa668ee7cdb3R10-R31 not neeeded since managed by kres